tests/test_colocateddata.py

```diff
@@ -23,9 +24,9 @@ def test_ColocatedData__init__(data: Path | str | ArrayLike):
 @pytest.mark.parametrize(
     "data,exception",
     [
-        (None, AttributeError),
-        ("Blaaaaa", IOError),
-        (np.ones(3), DataDimensionError),
+        # (None, AttributeError),
```

Comment:
Maybe test that exceptions are raised consistently by data-types (ie. xarray and numpy array).

Reply:
Added a test